src/github/mod.rs

```diff
@@ -13,11 +13,14 @@ pub struct Repository {
     pub repo: String,
 }
 
-pub fn latest_release(repository: &Repository) -> Result<Release, GithubError> {
+pub fn get_release(repository: &Repository, tag: Option<&Tag>) -> Result<Release, GithubError> {
```

Comment:
You forgot to update all usages of `latest_release(repository)` to `get_release(repository, None)`.
If we don't do that the build is broken on this commit :)

Reply:
Done, please review again.